new-dti-website-redesign/src/components/TeamCard.tsx

```diff
@@ -0,0 +1,136 @@
+import { getColorClass, productLinks } from '@/utils/memberUtils';
+import Image from 'next/image';
+import Link from 'next/link';
+
+const MemberSummary = ({
+  user,
+  image,
+  enlarged = false
+}: {
+  user: IdolMember;
+  image: string;
+  enlarged?: boolean;
+}) => (
+  <>
+    <Image
+      src={image}
+      alt={`${user.firstName} ${user.lastName}`}
```

Comment:
```suggestion
      alt={`${user.firstName} ${user.lastName}'s profile picture`}
```

(just to make the alt text a bit more descriptive)